ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -61,3 +82,32 @@ func readECSParams(filename string) (*ECSParams, error) {
 
 	return ecsParams, nil
 }
+
+func ConvertToEcsNetworkConfiguration(ecsParams *ECSParams) *ecs.NetworkConfiguration {
+	networkConfig := ecsParams.RunParams.NetworkConfiguration
+	awsvpcConfig := networkConfig.AwsVpcConfiguration
+	subnets := awsvpcConfig.Subnets
+	securityGroups := awsvpcConfig.SecurityGroups
+
+	ecsSubnets := make([]*string, len(subnets))
+	for i, subnet := range subnets {
```

Comment:
can you also verify that `len(subnets) > 0`

Reply:
done